deploy/database

```diff
@@ -0,0 +1,13 @@
+set -e
+
+echo "
+    CREATE DATABASE $Database;
+" | psql --host=$DBAddress --username=$DBUsername --port=5432
```

Comment:
@Rub21 I had to remove `--dbname=$Database` from this line to make this script to work. We should probably check the deploy/install script and see if we get the same error as I had. Basically you don't reference $Database on the CREATE DATABASE command, because the DB does not exist yet.